hazelcast/src/main/java/com/hazelcast/jet/impl/execution/init/JetInitDataSerializerHook.java

```diff
@@ -104,6 +105,7 @@ public final class JetInitDataSerializerHook implements DataSerializerHook {
     public static final int SNAPSHOT_STATS = 30;
     public static final int PREPARE_FOR_PASSIVE_CLUSTER_OP = 31;
     public static final int SNAPSHOT_VALIDATION_RECORD = 32;
+    public static final int ADD_JOB_STATUS_LISTENER_OP = 33;
```

Comment:
we were supposed not to reuse these ids

Reply:
We may not be supposed to reuse them, but I don't want to pollute the id space quickly. After all, we don't provide rolling upgrade compatibility guarantees for Jet except for patch releases.

Reply:
In the specific case it can be ok to reuse this ID (they were removed in 5.0 with the merge of IMDG + Jet), but in general it's not a good practice because it can lead to confusing behaviour.